wordpress-mobile/WordPress-Android
# Dashboard Cards: Pages handle click actions

Closes #18216 

This PR handles the Pages item click actions
- Track the page item tap event
- Navigate user to the correct tab in the pages view
- Created two new NavigationActions `OpenPagesDraftsTab` and `OpenPagesScheduledTab`
- Update Pages to support opening to a specific tab
- Updated and wrote new unit tests

**To test:**

Setup
1. Install the Jetpack app (do a fresh install - want to make sure the DB is clear)
2. Login with a wp account
3. Select a site in which you have access to manage content/admin
4. Create the following pages: Draft, Published, and Scheduled
5. Navigate to App Setting > Debug Settings 
6. Enable `dashboard_card_page` and restart the app

**Tapping on a Draft page item within the card navigates to Pages "Draft" tab **
1. Navigate to Home tab
2. Scroll to the pages card 
3. Tap on the Drafts page item 
4. ✅ Verify you are taken to the Pages "draft" tab
5. ✅ Verify log contains: 🔵 Tracked: my_site_dashboard_card_item_tapped, Properties: {"type":"pages","subtype":"draft"}
----------------
**Tapping on a Scheduled page item within the card navigates to Pages "Scheduled" tab **
1. Navigate to Home tab
2. Scroll to the pages card 
3. Tap on the Scheduled page item 
4. ✅ Verify you are taken to the Pages "scheduled" tab
5. ✅ Verify log contains: 🔵 Tracked: my_site_dashboard_card_item_tapped, Properties: {"type":"pages","subtype":"scheduled"}
---------------
**Tapping on a Published page item within the card navigates to Pages "Published" tab **
1. Navigate to Home tab
2. Scroll to the pages card 
3. Tap on the Published page item 
4. ✅ Verify you are taken to the Pages "published" tab
5. ✅ Verify log contains: 🔵 Tracked: my_site_dashboard_card_item_tapped, Properties: {"type":"pages","subtype":"published"}

## Regression Notes
1. Potential unintended areas of impact
The pages click items do not navigate properly

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual and unit testing

3. What automated tests I added (or what prevented me from doing so)
Add tests to `CardsTrackerTest` and `MySiteViewModelTest`

PR submission checklist:


UI Changes testing checklist: N/A
